LoopBack/LBFile.m

```diff
@@ -5,6 +5,9 @@
  * @copyright (c) 2014 StrongLoop. All rights reserved.
  */
 
+#if TARGET_OS_IPHONE
```

Comment:
Just a quick note -- we needed this `#if TARGET_OS_IPHONE` to avoid the OS X build to break.